Nikita_Korzhavin/api/calculateSmthInArray.js

```diff
@@ -0,0 +1,13 @@
+function calculateSmth(param, array) {
```

Comment:
This function name is not much meaningful

Reply:
fixed